specification/configuration.md

```diff
@@ -130,6 +136,27 @@ beyond the OpenTelemetry specification exist.
   - Distribution MUST default to `otlp`
   - Distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
 
+### Real User Monitoring Libraries
+
+Real User Monitoring (RUM) instrumentation libraries cannot use environment variables for configuration.
+Instead, they MUST expose a `SplunkRum` object/class/namespace (depending on the language used) that allows
+setting the following properties (in a language-specific way: Java may use builders, Swift can use named
+parameters with default values, JavaScript can use objects, etc.):
+
+| Configuration property (default value) | Description                                                                                                                                                                                                 |
+| -------------------------------------- | -----------                                                                                                                                                                                                 |
+| `beaconUrl` ()                         | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. [1]                                                                                                                                   |
+| `rumAccessToken` ()                    | RUM authentication token. [1]                                                                                                                                                                               |
```

Comment:
We use the term `rumAuth` for this.

Reply:
Yes, but should we continue to use `rumAuth`? `rumAccessToken` is more similar to `SPLUNK_ACCESS_TOKEN` that is used in collector & instrumentation libs, and it is an "access token" (I mean you can find it on the "Access Tokens" page in the UI)

Reply:
It is named differently because it is different in kind.  `rumAuth` is public and we don't want anybody pasting a random other access token into that space.

Reply:
Fair point -- I'll revert to `rumAuth`.